### ISSUE TITLE
SyntaxError when running Prettier: Property or signature expected.

### ISSUE THREAD

**Issue by JanMalch** [drwpow/openapi-typescript#463]
I'm getting a `SyntaxError: Property or signature expected.` when Prettier formats the TypeScript output for certain YAMLs.

```
npx openapi-typescript docs/api.yaml --output schema.ts
✨ openapi-typescript 3.0.0
� Loading spec from docs/api.yaml…
SyntaxError: Property or signature expected. (154:1)
  152 |   }
  153 |   requestBody: components["requestBodies"]["Pet"];
> 154 | ;
      | ^
  155 |   }
  156 |   "addPet": {
  157 |       responses: {
    at e (XXX\node_modules\prettier\parser-typescript.js:1:322)
    ...
```

This error only occurs when using [this `petstore.yaml` from OpenAPITools](https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml).
The [`petstore.yaml` from OAI](https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml) works as expected.

**Comment by drwpow** [comment]
Good find! Will take a closer look soon, thank you. Thanks for posting the erring schema as well. That’s super helpful.

**Comment by JanMalch** [comment]
It seems that adding a `requestBody` and referencing it causes issues. I just added [one from the breaking yaml](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml#L44-L45) to the working yaml and it breaks it as well.

But when inlining the `requestBody` it works:

```yaml
post:
  summary: Create a pet
  operationId: createPets
  tags:
    - pets
  responses:
    '201':
      description: Null response
    default:
      description: unexpected error
      content:
        application/json:
          schema:
            $ref: "#/components/schemas/Error"
  # added this, original working yaml didn't even have a requestBody
  requestBody:
    description: Pet object that needs to be added to the store
    required: true
    content:
      application/json:
        schema:
          $ref: '#/components/schemas/Pet'
```

Hope that helps.

**Comment by drwpow** [comment]
Fixed and released `3.0.1`. The problem wasn’t simply a $ref in `responseBody`, because that was tested. It was a special feature we have where we make `operationId` accessible via `operations["myId"]`. There was a typo in that, and it was generating differently. Anyway that `operationId` was the difference between those 2 petstore schemas that was throwing an error.

Please give the new version a try; hopefully that fixes the issue!
